CHANGELOG.yml

```
@@ -20,6 +20,7 @@ upcoming:
     - Add order history to user profile (behind feature flag) - alexj105
     - Add order history skeleton (behind feature flag) - Serge0n
     - Add orders to order history (behind feature flag) - Serge0n
+    - Add order details page (app registry, a route with orderID, relay container)
```

```suggestion
    - Add order details page (app registry, a route with orderID, relay container) - irina-sid
```
minor suggestion to add username just so we know who to reach out to about the change

---

Sure!

---

Sure!